### PR TITLE
Reorganise FAIR2RS landing page

### PR DESCRIPTION
**Pull request by tdjames1**

### CODE REVIEW

File: pages/training/fair4rs/index.md [hunk 1]
```diff
@@ -5,12 +5,23 @@ type: text
 permalink: /training/fair4rs/
 ---
 
+## Introduction and target audience
 
-## What are the FAIR principles for research software?
+This training offers a modular programme to support researchers in applying FAIR (Findable, Accessible, Interoperable,
```
Comment (Robadob) [inline pages/training/fair4rs/index.md]:
```suggestion
This training curriculum offers a modular programme to support researchers in applying FAIR (Findable, Accessible, Interoperable,
```

"This training" feels like a word is missing.

File: pages/training/fair4rs/index.md [hunk 2]
```diff
@@ -5,12 +5,23 @@ type: text
...
+Reusable) principles and open research practices to their research software. The overview module provides an
+introduction to [FAIR for research software](#what-are-the-fair-principles-for-research-software) and gives an overview
+of the topics that are offered in more detail in [additional modules](#outline-of-the-programme), which can be selected
+according to an individual’s learning goals.
 
-The FAIR principles were originally developed as guidelines to enhance the reusability of research data. The FAIR
-principles apply the concepts of Findability, Accessibility, Interoperability and Reusability to scholarly data
-holdings, and were also intended to be applied to other digital research objects such as algorithms and workflows[^1].
+This programme is aimed at researchers, including PhDs and postgraduate research students, who create code (whether a few
```
Comment (Robadob) [inline pages/training/fair4rs/index.md:16]: > including PhDs and postgraduate research students

I've always considered these to be equivalent, I wasn't aware the university offered MRes as an option (other than if you drop out of your PhD at 12 months). As a minimum PhDs are postgraduate research students.

Perhaps reword to research staff and students?

File: pages/training/fair4rs/index.md [hunk 3]
```diff
@@ -5,12 +5,23 @@ type: text
...
+scripts or something more substantial) as part of their research and who want to make their research more open by
+applying the FAIR principles to their software.
```
Comment (Robadob) [inline pages/training/fair4rs/index.md]:
```suggestion
applying the FAIR principles to their software or simply want to become more confident in the research code they are writing.
```

Broadens the appeal slightly

**Review by Robadob — APPROVED** [review]
Looks good, everything besides my first comment is picky.

*I haven't checked any of the new internal page hyperlinks.*

File: pages/training/fair4rs/index.md (hunk 2 above)
Comment (Romain-Thomas-Shef) [reply pages/training/fair4rs/index.md:16]: I kind of prefer the version as it is.

**Review by Romain-Thomas-Shef — APPROVED** [review]
All good on my side.

File: pages/training/fair4rs/index.md (hunk 2 above)
Comment (tdjames1) [reply pages/training/fair4rs/index.md:16]: I did an MRes at this university!

**Comment by tdjames1** [comment]
> _I haven't checked any of the new internal page hyperlinks._

Thanks for that prompt, fixed one that failed due to and/& difference!

**Pull request merged**